src/pages/taskHistory/TaskHistoryRow.tsx

```diff
@@ -38,6 +42,9 @@ const TaskHistoryRow = memo((props: ListChildComponentProps) => {
               failingTests={failingTests}
               label={label}
               loading={loading}
+              sendAnalytics={(v: TaskCellAnalytics) => {
+                sendEvent({ name: "Click task history task cell", ...v });
```

Comment:
Maybe we call this event `Click Task Cell` and have the specific page be an option. That way we can query for all task cell clicks.